src/tools/tests/golden_kt.arcs

```diff
@@ -19,5 +19,23 @@ particle KotlinPrimitivesGolden
     flt: Float,
     dbl: Double,
     txtlst: List<Text>,
-    lnglst: List<Long>
+    lnglst: List<Long>,
```

Comment:
FYI I'm slowly working to restructure the Kotlin codegen into smaller components and decrease the reliance on golden files for testing. Over reliance on goldens have bitten us a number of times, as they don't encourage testinf the full range of cases. Goldens should just showcase how things are assembled together into a file.

There are now a number of files containing unit tests for kotlin code generation, for this PR the relevant ones would be:
kotlin-schema-generator-test.ts
kotlin-entity-generator-test.ts

Please try adding unit tests instead of growing goldens.